lib/parsec/fileparse.py

```diff
@@ -259,11 +241,8 @@ def read_and_proc(fpath, template_vars=None, viewcfg=None, asedit=False):
 
     # inline any cylc include-files
     if do_inline:
-        try:
-            flines = inline(
-                flines, fdir, fpath, False, viewcfg=viewcfg, for_edit=asedit)
-        except IncludeFileNotFoundError as exc:
-            raise FileParseError(str(exc))
```

Comment:
One example of catching a specific exception then raising a generic one in its place.